src/ol/map.js

```diff
@@ -1486,7 +1486,7 @@ ol.Map.createOptionsInternal = function(options) {
       logos[logo] = '';
     } else if (logo instanceof HTMLElement) {
       logos[goog.getUid(logo).toString()] = logo;
-    } else if (goog.isObject(logo)) {
+    } else if (logo !== null && typeof logo === 'object') {
```

Comment:
A truthy check for logo should suffice here as well. Then you'll also probably have to add a type cast to logo to satisfy the compiler.